__tests__/core/generate.test.ts

```diff
@@ -27,9 +27,16 @@ describeAllImplementations((implementation) => {
         updateStaleOnly: false,
         logLevel: "verbose",
         outputFolder: null,
+        aliases: {
+          "~fancy-import": "complex",
+          "~another": "style",
+        },
+        aliasPrefixes: {
+          "~": "nested-styles/",
+        },
       });
 
-      expect(fs.writeFileSync).toHaveBeenCalledTimes(6);
+      expect(fs.writeFileSync).toHaveBeenCalledTimes(9);
```

Comment:
This number is higher because more files are now compiled, since the tool forces you to handle build errors. This is also why setting the `aliases` and `aliasPrefix` was required for this test.